misc/Dockerfile

```diff
@@ -23,4 +23,4 @@ RUN pip3 install -r misc/requirements.txt
 
 ENV INDEXER_DATA="${HOME}/indexer/"
 # Run test script
-ENTRYPOINT ["/bin/bash", "-c", "sleep 5 && python3 misc/e2elive.py --connection-string \"$CONNECTION_STRING\" --indexer-bin /opt/go/indexer/cmd/algorand-indexer/algorand-indexer --indexer-port 9890"]
+ENTRYPOINT ["/bin/bash", "-c", "sleep 5 && python3 misc/e2elive.py --connection-string \"$CONNECTION_STRING\" --s3-source-net \"$CI_E2E_FILENAME\" --indexer-bin /opt/go/indexer/cmd/algorand-indexer/algorand-indexer --indexer-port 9890"]
```

Comment:
Using same naming convention `CI_E2E_FILENAME ` for the S3 network artifact as in [go-algorand](https://github.com/algorand/go-algorand/blob/cdd4aa1305fd3d99c83c7a629f2202ffe0f3964e/test/scripts/e2e.sh#L184).